app/models/submission_entry.rb

```diff
@@ -5,6 +5,7 @@ class SubmissionEntry < ApplicationRecord
   belongs_to :submission_file, optional: true
 
   validates :data, presence: true
+  validates :entry_type, inclusion: { in: %w[invoice order] }, allow_blank: true
 
   scope :sheet, ->(sheet_name) { where("source->>'sheet' = ?", sheet_name) }
```

Comment:
Is it worth changing, or making a card to update this scope to use the `entry_type` column instead?

Reply:
I was planning to do that as a separate PR once: a) the ingest is updated to set the new column; and b) the data migration to backfill the data has been run.